smalltalksrc/VMMakerTests/VMPrimitiveTest.class.st

```diff
@@ -953,6 +1393,17 @@ VMPrimitiveTest >> testPrimitiveGetImmutabilityReturnsTrueIfObjectIsImmutable [
 
 { #category : #'tests - primitiveGreaterOrEqual' }
 VMPrimitiveTest >> testPrimitiveGreaterOrEqualFailsWithFloat [
+	
+	|classFloat|
+
+	classFloat := self newClassInOldSpaceWithSlots: 0 instSpec: memory firstLongFormat.
+	memory setHashBitsOf: classFloat to: ClassFloatCompactIndex.
+	memory
+		storePointer: ClassFloatCompactIndex
+		ofObject: memory classTableFirstPage
+		withValue: classFloat.
+		
+	"This simulated classFloat class is necessary because the 32bits VM cannot instanciate boxed floats by itself"
```

Comment:
Maybe all this code could be extracted in a third method `createBoxedFloatClass` or something like that. That would help in not duplicating this code :)